subsys/net/lib/nrf_cloud/include/nrf_cloud_mem.h

```diff
@@ -12,31 +12,31 @@ extern "C" {
 #endif
 
 #include <zephyr/kernel.h>
+#include <net/nrf_cloud_os.h>
 
-/**@brief Method to allocate zero-initialized memory of size COUNT*SIZE, for
+/**@brief Method to allocate zero-initialized memory of size count*size
  * internal use in the module.
  *
- * @param[in] COUNT Count of memory blocks requested.
- * @param[in] SIZE Size of memory block.
+ * @param[in] count Count of memory blocks requested.
+ * @param[in] size Size of memory block.
  *
  * @retval A valid pointer on SUCCESS, else, NULL.
  */
-#define nrf_cloud_calloc(COUNT, SIZE) k_calloc(COUNT, SIZE)
+void *nrf_cloud_calloc(size_t count, size_t size);
```

Comment:
I'm thinking if these functions should be inline functions but someone else should comment on that too.

Reply:
Inline would probably be better. I was thinking whether those could be even defines, but then the struct for the pointers would have to be global, I guess.

Reply:
As discussed, inline/defines for this does not work.